pkg/api/controller.go

```diff
@@ -1532,9 +1533,12 @@ func (c *Controller) ListRunHooks(w http.ResponseWriter, r *http.Request, reposi
 			StartTime: val.StartTime,
 			EndTime:   &val.EndTime,
```

Comment:
This is also true before your PR - but because the task didn't start and end time is optional, we should keep it nil.